Assets/Scripts/PlayerManager.cs

```diff
@@ -22,14 +22,23 @@ public bool CanRest
 
     private void Update()
     {
-        if (CanRest && Input.GetKey(KeyCode.E))
+        if (CanRest)
         {
             bool isResting = Input.GetKey(KeyCode.E);
+            UIManager.Instance.UpdateRestingText(isResting);
+
             if (isResting)
             {
                 HandleIncreaseStamina();
```

Comment:
Does this mean that player can just rest and not play the minigame to restore stamina?

Reply:
Yea right now there's a choice of, E to rest, P to play a minigame.
Was thinking maybe keep both for now for debugging purposes and because the design isn't finalized yet?